docs/openapi.yaml

```diff
@@ -270,6 +270,47 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
+  /candidate/publication/{candidatePublicationId+}:
+    get:
+      description: 'Get scientific index candidate by publication identifier'
+      parameters:
+        - in: path
+          name: candidatePublicationId+
+          description: The UTF-8 Encoded pubication identifer of the candidate
```

Comment:
Is the type of path parameter candidatePublicationId a URI or an UUID identifier? If it is a URI, make it clearer? (You are testing with a URI in FetchNviCandidateByPublicationIdHandlerTest)